src/utils.js

```diff
@@ -69,6 +69,27 @@ export const setQueryParam = (param, value) => {
   localStorage.savedQueryParams = qs.toString()
 }
 
+export const appendQueryParam = (param, value) => {
+  const qs = new URLSearchParams(window.location.search)
+  if (qs.get(param)) {
+    qs.set(param, [...new Set([...qs.get(param).split(','), value])].join(',')) // remove duplication
+  } else {
+    qs.set(param, value)
+  }
+  window.history.replaceState(null, '', '?' + qs.toString())
+  localStorage.savedQueryParams = qs.toString()
+}
```

Comment:
Could we write this in terms of `setQueryParam`?
Something like:
```js
export const appendQueryParam = (param, value) => {
  const existing = (qs.get(param) ?? []).split(',')
  setQueryParam(param, [...new Set([...existing, value])].join(','))
}
```

Reply:
This leaves a comma in front of the value, rewrote to take that into account